doc/classes/Node.xml

```diff
@@ -578,235 +608,234 @@
+			The owner of this node. When saving the owner node in a [PackedScene], all the nodes it owns are also saved with it. In the editor, nodes not owned by the scene root are not displayed in the Scene dock, and will [b]not[/b] be saved. To prevent this, remember to set the owner after calling [method add_child].
+			[b]Note:[/b] The owner does [i]not[/i] need to be an ancestor of this node. For simplicity, the rest of the documentation may not account for this.
 		</member>
 		<member name="process_mode" type="int" setter="set_process_mode" getter="get_process_mode" enum="Node.ProcessMode" default="0">
-			Can be used to pause or unpause the node, or make the node paused based on the [SceneTree], or make it inherit the process mode from its parent (default).
+			The node's processing behavior. Can be changed to pause the [SceneTree]'s processing calls for this node. See [enum ProcessMode].
 		</member>
 		<member name="process_priority" type="int" setter="set_process_priority" getter="get_process_priority" default="0">
-			The node's priority in the execution order of the enabled processing callbacks (i.e. [constant NOTIFICATION_PROCESS], [constant NOTIFICATION_PHYSICS_PROCESS] and their internal counterparts). Nodes whose process priority value is [i]lower[/i] will have their processing callbacks executed first.
+			The node's execution order of the process callbacks ([method _process], [method _physics_process], and internal processing). Nodes whose priority value is [i]lower[/i] call their process callbacks first, regardless of tree order.
 		</member>
 		<member name="scene_file_path" type="String" setter="set_scene_file_path" getter="get_scene_file_path">
-			If a scene is instantiated from a file, its topmost node contains the absolute file path from which it was loaded in [member scene_file_path] (e.g. [code]res://levels/1.tscn[/code]). Otherwise, [member scene_file_path] is set to an empty string.
+			The original scene's file path, if the node has been instantiated from a [PackedScene] file. Only scene root nodes contains this.
 		</member>
 		<member name="unique_name_in_owner" type="bool" setter="set_unique_name_in_owner" getter="is_unique_name_in_owner" default="false">
-			Sets this node's name as a unique name in its [member owner]. This allows the node to be accessed as [code]%Name[/code] instead of the full path, from any node within that scene.
-			If another node with the same owner already had that name declared as unique, that other node's name will no longer be set as having a unique name.
+			If [code]true[/code], the node can be accessed from any node sharing the same [member owner] or root node, with special [code]%Name[/code] syntax in [method get_node].
```

Comment:
I'm not sure about "sharing the same owner or root node".

1. Their owner is the same, or their root node is the same: it's only about owner.
2. Their owner is the same, the owner means root node: owner does not have to be a root node.